### PR TITLE
Fix update vendors

### PR DESCRIPTION
**Pull request by kevin-wangzefeng**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:
verify vendors and update vendors scripts currently don't check go.mod and go.sum files in staging repos. This PR is to fix it.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/update-vendor.sh [hunk 1]
```diff
@@ -34,6 +34,14 @@ function kubeedge::util::list_staging_repos() {
...
+for repo in $(kubeedge::util::list_staging_repos); do
+  pushd "${KUBEEDGE_ROOT}/staging/src/github.com/kubeedge/${repo}"
+  echo "running 'go mod tidy' for ${repo}"
+  go mod tidy
+  popd
+done
+
+
 echo "running 'go mod tidy'"
 go mod tidy
```
Comment (chendave) [inline hack/update-vendor.sh]: why run this two times?

Comment (fisherxu) [reply hack/update-vendor.sh]: I think that `go mod tidy` in for loop(L40) generated go module files for projects under staging(beehive&viaduct). @chendave

**Pull request merged**